sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -925,6 +925,26 @@ class DDLSuite extends QueryTest with SharedSQLContext with BeforeAndAfterEach {
     }
   }
 
+  test("show table extended ... partition") {
```

Comment:
Yes it works, but it outputs the absolute path for `Location`, so the test suite will fail on another environment.